tests/python/contrib/test_hexagon/test_launcher.py

```diff
@@ -40,6 +42,251 @@
 # triggering TIME_WAIT state on the server socket. This prevents another
 # server to bind to the same port until the wait time elapses.
 
+@requires_hexagon_toolchain
+def test_add_hvx(android_serial_number, tvm_tracker_host, tvm_tracker_port, adb_server_socket):
+    """
+    Starting with an elementwise-add computation, try various schedules / optimizations to
+    see the impact they have on performance.
+
+    The main motivation for this test is to explore the relationship between these
+    schedules / optimizations vs. how effectively the primfunc uses the Hexagon's
+    HVX units.
+    """
+
+    host_output_dir = tempfile.mkdtemp()
```

Comment:
Does the `host_output_dir` get cleaned up after a run?  If not, this could cause the temporary directories to balloon up.  Could we add a fixture with something like the following, so that the benchmark directory gets cleaned up unless an environment variable is set.

```python
@pytest.fixture(scope="session")
def tvm_benchmark_path():
    if 'TVM_BENCHMARK_PATH' in os.environ:
        yield os.environ['TVM_BENCHMARK_PATH']
    else:
        with tempfile.TemporaryDirectory(prefix='tvm_hexagon_benchmarks_') as temp_dir:
            yield temp_dir
```

Reply:
I intentionally want the output files to stay around, because one of the script's uses is to investigate correlations between code-structure and performance.

I think the real issue here is that `test_add_hvx` isn't really a unit test at this point.  As [mentioned in another comment](https://github.com/apache/tvm/pull/10604#discussion_r834690032), I think the best solution is to move this functionality into a separate, non-unit-test script.

Reply:
That's a good call, and avoids the risk of it being caught in the CI unintentionally, if the environment changes in the fture.